src/index.tsx

```diff
@@ -48,17 +48,11 @@ export {
 export {
   LanguageMenu,
   LanguageMenuProps,
-} from './core/LanguageMenu/LanguageMenu';
-export {
   LanguageMenuItem,
   LanguageMenuItemProps,
   LanguageMenuLink,
   LanguageMenuLinkProps,
-  LanguageMenuItemLanguage,
-  LanguageMenuItemLanguageProps,
-  LanguageMenuLinkLanguage,
-  LanguageMenuLinkLanguageProps,
```

Comment:
These are no longer exported as those can be used directly from `LanguageMenuItem, LanguageMenuItemProps` and `LanguageMenuLink, LanguageMenuLinkProps`.